### DISCUSSION TITLE
No-married, unions, partnerships, biological parents...

### DISCUSSION THREAD

**Discussion by maco1717** [fisharebest/webtrees#4830] [General]
Is this supported by the application by any chance?

I can only see family with marriages. which is not very 21st century.

**Comment by fisharebest** [comment]
Yes.

You can add married/unmarried husband/wife/same-sex partner.

Rather that create a dozen edit options, there is simply "add a husband/wife", as this is the most common option.

But on the next page, you'll be able to select the sex, create a marriage or not, etc.

The only real limitation is that a family cannot contain more than two parents.  So, polygamy/polyandry doesn't work.  It's a limitation of the GEDCOM data format.  The GEDCOM maintainers are working on this for a future version.  When they support it, we will too.
